src/Core/Plugin.php

```diff
@@ -170,6 +184,24 @@ public static function load($plugin, array $config = [])
     }
 
     /**
+     * Loads the config mappings for plugins.
+     *
+     * The map data in CONFIG/plugins.php map data is used set default paths
+     * for where plugins are located.
+     *
+     * @return void
+     */
+    protected static function _loadConfigMap()
+    {
+        if (!file_exists(CONFIG . 'plugins.php')) {
+            static::$_config = [];
+            return;
+        }
+        $config = include CONFIG . 'plugins.php';
```

Comment:
`composer show -i` gives you listed of installed packages and `composer show <package-name>` gives you info about the package which includes the package type. But these commands seem to take quite some time to show output. So even if we were to use them we would have to cache the info, so I would rather just directly use a config file.